syntax="proto2";
package AccountPacket;

message S_Account{
	required uint32 uid=1;
	required uint32 hp=2;
	required uint32 x=3;
	required uint32 y=4;
}